### PR TITLE
fix issue 38229

### PR DESCRIPTION
**Pull request by wanghaoran1988**
fix this https://github.com/kubernetes/kubernetes/issues/38220

### CODE REVIEW

**Comment by ixdy** [comment]
@k8s-bot ok to test

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -5019,3 +5019,11 @@ func GetNodeExternalIP(node *v1.Node) string {
 	}
 	return host
 }
+func getNodeNames(nodes []*v1.Node) []string {
+	names := []string{}
+	for _, node := range nodes {
+		names = append(names, node.Name)
+	}
+	return names
+
```
Comment (k82cn) [inline test/e2e/framework/util.go]: remove empty line.

Comment (wanghaoran1988) [inline test/e2e/framework/util.go]: > remove empty line.
@k82cn ok

**Comment by wanghaoran1988** [comment]
issue already resolved, close.

**Comment by lavalamp** [comment]
Sorry for not responding, I was OOO.

**Pull request closed without merging**